projects/httpcomponents-client/project.yaml

```diff
@@ -11,3 +11,4 @@ vendor_ccs:
 - yakdan@code-intelligence.com
 - glendowne@code-intelligence.com
 - hlin@code-intelligence.com
+- yoshi.weber@gmail.com
```

Comment:
In the future can you add yourself to `auto_ccs` since youre not a vendor but a maintainer of these projects?

Reply:
Sure

Reply:
To clarify: Can you put your email under

```yaml
auto_ccs:
- yoshi.weber@gmail.com
vendor_ccs:
...
```

Thanks!